src/Client.php

```diff
@@ -4,62 +4,103 @@
 
 use Exception;
```

Comment:
I think we can simplify farther. Drop the `Duplicate` exception. Add a new generic `/src/Exception.php` and pass the original error message and code we get from the server. We can catch it later in Utopia/Database and throw a different exception according to our needs.